docker/docker-run.sh

```diff
@@ -20,8 +20,9 @@ docker run --rm \
     -v $WS_DIR:/home/$USER/catkin_ws \
     -v /tmp/.X11-unix:/tmp/.X11-unix \
     `[ -f ~/.gitconfig ] && echo "-v $HOME/.gitconfig:/home/$USER/.gitconfig:ro"` \
+    -v $HOME/.Xauthority:/home/$USER/.Xauthority \
```

Comment:
This is not portable to Mac. We already do this exact thing on line 19, but do it as read only, which I'd prefer unless if there is a need to be able to change it.